code/game/gamemodes/vampire/vampire_other.dm

```diff
@@ -0,0 +1,43 @@
+/*
+	In this file:
+		various vampire interactions and items
+*/
+
+
+/obj/item/clothing/suit/draculacoat
+	name = "Vampire Coat"
+	desc = "What is a man? A miserable little pile of secrets."
+	alternate_worn_icon = 'icons/mob/suit.dmi'
+	icon = 'icons/obj/clothing/suits.dmi'
+	icon_state = "draculacoat"
+	item_state = "draculacoat"
+	body_parts_covered = CHEST|GROIN|LEGS|ARMS
+	flags_inv = HIDEJUMPSUIT
+	armor = list(melee = 30, bullet = 20, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0)
+
+/mob/living/carbon/human/handle_fire()
```

Comment:
snowflake, AND defined outside of human files

Reply:
@MoreRobustThanYou STOP FUCKING DEFINING THIS SHIT OUT OF HUMAN FILES YOU KNOW HOW PAINFUL IT IS IF BUGS SPRING UP LATER OR IF THIS FUCKS UP?!